docs/src/models.md

```diff
@@ -0,0 +1,15 @@
+# Predefined Models
+Predefined agent based models exist in the `Models` submodule in the form of functions that return `model, agent_step!, model_step!` when called.
+They are the versions already explained in the Examples section of the docs.
```

Comment:
> They are the versions already explained in the Examples section of the docs.
Perhaps move this underneath the access example, and reword to:
The Examples section of the docs outline how to use and interact with each model.